velox/common/memory/MappedMemory.h

```diff
@@ -498,6 +517,14 @@ class MappedMemory : public std::enable_shared_from_this<MappedMemory> {
   inline static std::atomic<uint64_t> totalSmallAllocateBytes_;
   inline static std::atomic<uint64_t> totalSizeClassAllocateBytes_;
   inline static std::atomic<uint64_t> totalLargeAllocateBytes_;
+
+ private:
+  // Singleton instance.
+  inline static std::shared_ptr<MappedMemory> instance_;
```

Comment:
This is an anti pattern. A better way is to use a static local variable in a method.